README.rst

```diff
@@ -45,4 +46,8 @@ To build the pdf form of the documentation simply execute the following command:
 
   make latexpdf
 
-Note: Creating the pdf form of the documentation requires that latexmk is available.
+Note: Creating the pdf form of the documentation requires that latexmk is available, information on latexmk can be found at https://ctan.org/pkg/latexmk/.
+
+Note: The documentation built with `make html` is different to the documentation built with `make latexpdf`.
+The html form of the documentation is the full form being both the normative specification and the informative specification.
+Whilst the pdf form of the documentation is solely the normative specification.
```

Comment:
PDF here too?
Also, either make this part of the sentence above, or ditch the "Whilst" and start with "The".